### PR TITLE
chore(project management): Pull Request Template

### PR DESCRIPTION
**Pull request by kuhnchris**
Attempt 2 of #490  - this time based on master.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Please make sure to check the following tasks before opening and submitting a PR
+
+* [ ] I understood and followed the [contribution guide](https://github.com/revoltchat/revolt/discussions/282)
+* [ ] I tested my change locally and it is working as intended
+* [ ] The change doesn't have any notable side effects on other project
```
Comment (Rexogamer) [inline .github/pull_request_template.md:5]: Minor typo:
```suggestion
* [ ] The change doesn't have any notable side effects on other Revolt projects
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## Please make sure to check the following tasks before opening and submitting a PR
+
+* [ ] I understood and followed the [contribution guide](https://github.com/revoltchat/revolt/discussions/282)
```
Comment (Rexogamer) [inline .github/pull_request_template.md:3]: Tense change:
```suggestion
* [ ] I understand and have followed the [contribution guide](https://github.com/revoltchat/revolt/discussions/282)
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+* [ ] I tested my change locally and it is working as intended
```
Comment (Rexogamer) [inline .github/pull_request_template.md:4]: Tense change:
```suggestion
* [ ] I have tested my change locally and it is working as intended
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+* [ ] The change doesn't have any notable side effects on other project
+* [ ] (optional) I opened a pull request on the [translation repository](https://github.com/revoltchat/translations)
```
Comment (Rexogamer) [inline .github/pull_request_template.md:6]: Tense chnage:
```suggestion
* [ ] (optional) I have opened a pull request on [the translation repository](https://github.com/revoltchat/translations)
```

**Review by Rexogamer — COMMENTED** [review]
A few more nitpicks (mainly consistent tensing) and this should be good to go:

**Comment by kuhnchris** [comment]
Thanks again - added it to the commit.

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+## Please make sure to check the following tasks before opening and submitting a PR
+
+* [ ] I understand and have followed the [contribution guide](https://github.com/revoltchat/revolt/discussions/282)
+* [ ] I have tested my change locally and it is working as intended
```
Comment (Rexogamer) [inline .github/pull_request_template.md:4]:
```suggestion
* [ ] I have tested my changes locally and they are working as intended
```

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+* [ ] The change doesn't have any notable side effects on other Revolt projects
```
Comment (Rexogamer) [inline .github/pull_request_template.md:5]:
```suggestion
* [ ] These changes do not have any notable side effects on other Revolt projects
```

**Review by Rexogamer — COMMENTED** [review]
One last round and this should be perfect 🎉

**Comment by kuhnchris** [comment]
:partying_face:  :tada: thanks - since I'm far from a native english speaker I trust your changes there.
Thanks again and have a good one!

**Review by Rexogamer — APPROVED** [review]

**Pull request merged**